pkg/stats/bucket.go

```diff
@@ -4,14 +4,46 @@ import (
 	"fmt"
 	"net/http"
 	"net/url"
+	"strconv"
 	"strings"
 )
 
 const (
-	TotalRequestBucket   = "total.requests"
-	TotalRoundTripBucket = "total.round"
+	totalRequestBucket   = "total.requests"
+	totalRoundTripBucket = "total.round"
+	pathIDPlaceholder    = "-id-"
 )
 
+func testAlwaysTrue(string) bool {
+	return true
```

Comment:
This is test function stub for paths that always has ID at second level (most of paths actually). See `hasIDAtSecondLevel` bellow.